meal-planner-fredericton-ui/src/dashboard/Dashboard.jsx

```diff
@@ -153,6 +154,36 @@ function doDeleteMeapPlan({ id }) {
   });
 }
 
+const DeleteDialog = ({mealPlan, onDelete, onCancel}) => {
+  
+  
+  const handleConfirm = async () => {
+      await doDeleteMealPlan(mealPlan);
```

Comment:
The doDeleteMealPlan fn which connects to the postgraphile is called to delete the record from the database when button Yes is clicked. The mealPlan object is received as a prop and passed as an argument to doDeleteMealPlan.